$OpenBSD$
# Drop the invalid `forall` export and definition so constraints-0.13.4
# parses on GHC 9.10.
--- constraints-0.13.4/src/Data/Constraint/Forall.hs
+++ constraints-0.13.4/src/Data/Constraint/Forall.hs
@@ -34,7 +34,6 @@
   , Forall1, inst1
   , ForallT, instT
   , ForallV, InstV (instV)
-  , forall
   ) where

 import Data.Constraint
@@ -193,5 +192,3 @@
         Sub Dict -> case instV :: ForallV (p a) :- c of
             Sub Dict -> Dict

-forall :: forall p. (forall a. Dict (p a)) -> Dict (Forall p)
-forall d = case d :: Dict (p (Skolem p)) of Dict -> Dict
